lib/crypto/crypto.js

```diff
@@ -257,6 +257,33 @@ class Crypto {
       .join('.')
   }
 
+  /**
+   * Encrypt a private key using a public and private key.
+   *
+   * @param {string} privateKey
+   * @param {string} writerPrivateKey
+   * @param {string} readerPublicKey
+   *
+   * @return {Promise<string>}
+   */
+  async encryptPrivateKey(privateKey, writerPrivateKey, readerPublicKey) {
+    let rawPrivateKey = this.platform.b64URLDecode(privateKey)
+    let publicEncKey = this.platform.b64URLDecode(readerPublicKey)
+    let privateEncKey = this.platform.b64URLDecode(writerPrivateKey)
+
+    let nonce = await this.randomNonce()
+    let eak = await this.provider.encryptAsymmetric(
```

Comment:
nit: this is not an eak (encrypted access key), it's a private key, so let's update the var name to something like 'pk'.